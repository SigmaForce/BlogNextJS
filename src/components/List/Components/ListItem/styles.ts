import tw from 'tailwind-styled-components';

export const Container = tw.li`
    text-medium
    relative
    px-2
    font-light
    transition-colors
    duration-300
    hover:text-link
`;
