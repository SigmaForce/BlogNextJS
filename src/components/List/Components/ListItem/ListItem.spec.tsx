import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom';
import { ListItem } from './ListItem';

describe('<ListItem/>', () => {
  it('should render a ListItem', () => {
    const children = 'children';
    const { container } = render(<ListItem>{children}</ListItem>);
    const childrenElement = screen.getByText(children);
    expect(childrenElement).toBeVisible();
    expect(container).toMatchSnapshot();
  });
});
