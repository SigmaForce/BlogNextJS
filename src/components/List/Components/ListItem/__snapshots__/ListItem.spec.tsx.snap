// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ListItem/> should render a ListItem 1`] = `
<div>
  <li
    class="text-medium relative px-2 font-light transition-colors duration-300 hover:text-link"
  >
    children
  </li>
</div>
`;
